src/sentry/templates/sentry/js-sdk-loader.js.tmpl

```diff
@@ -47,63 +137,16 @@
     if (_oldOnunhandledrejection) _oldOnunhandledrejection.apply(_window, arguments);
   };
 
-  // Create a `script` tag with provided SDK `url` and attach it just before the first, already existing `script` tag
-  // Scripts that are dynamically created and added to the document are async by default,
-  // they don't block rendering and execute as soon as they download, meaning they could
-  // come out in the wrong order. Because of that we don't need async=1 as GA does.
-  // it was probably(?) a legacy behavior that they left to not modify few years old snippet
-  // https://www.html5rocks.com/en/tutorials/speed/script-loading/
-  var _currentScriptTag = _document.getElementsByTagName(_script)[0];
-  var _newScriptTag = _document.createElement(_script);
-  _newScriptTag.src = '{{ jsSdkUrl|safe }}';
-  _newScriptTag.crossorigin = 'anonymous';
-
-  // Once our SDK is loaded
-  _newScriptTag.addEventListener('load', function() {
-    try {
-      // Restore onerror/onunhandledrejection handlers
-      _window[_onerror] = _oldOnerror;
-      _window[_onunhandledrejection] = _oldOnunhandledrejection;
+  var attributes = Array.prototype.slice.apply(document.scripts).filter(function(script) {
```

Comment:
Let's move it to the very top, near `lazy` initialization. It can also be shortened to one for loop (dataset is supported everywhere, even old IE and androids).

```js
var lazy;

for (var i = 0; i < document.scripts.length; i++) {
  if (document.scripts[i].src.indexOf('{{ publicKey|safe }}') > -1) {
    lazy = document.scripts[i].dataset.lazy === 'yes';
    break;
  }
}
```